.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,57 +8,45 @@
 ## Related issue(s)
 
 <!---
-    This project only accepts pull requests related to open issues
-
     - If suggesting a new feature or change, please discuss it in an issue first.
     - If fixing a bug, there should be an issue describing it with steps to reproduce.
+    - If you're Adobe internal, add a Jira ticket number but do NOT link directly to Jira.
 -->
 
--
-
-## Motivation and context
-
-<!--- Why is this change required? What problem does it solve? -->
+-   fixes [Issue Number]
```

Comment:
@najikahalsema im forcing it in all PR templates now so its a no brainer, I don't mind it in front of Jira task numbers either.

Reply:
bless you!!!